packages/react-file-type-icons/package.json

```diff
@@ -26,6 +26,7 @@
     "@fluentui/eslint-plugin": "^1.0.0-beta.2",
     "@types/react": "16.9.42",
     "@types/webpack-env": "1.16.0",
+    "@types/jest": "~24.9.0",
```

Comment:
After this change was made, we changed master to pull this from the root instead
```suggestion
```